README.md

```diff
@@ -30,8 +30,13 @@ The `-v` option tells Docker to mount the home directory (`~`) to `/userhome` in
 ### Enabling graphics
 
 ##### Linux
-To use graphics, make sure you are in an X11 session and run the following command: 
-`docker run -e DISPLAY=$DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix --rm -it --user $(id -u) rootproject/root root`
+To use graphics, make sure you are in an X11 session and run the following command:
+
+```
+docker run -e DISPLAY=$DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix --rm -it --user $(id -u) rootproject/root root
+```
+
+On some platforms (e.g., Arch Linux) connections to the X server must be allowed explicitly by executing `xhost +` or an equivalent command.
```

Comment:
```suggestion
On some platforms (e.g., Arch Linux) connections to the X server must be allowed explicitly by executing `xhost local:root` or an equivalent command.
```

@amadio better? (I don't know what's a safer alternative)

Reply:
Yes. You could also consider linking to the Arch paged I mentioned in Mattermost.